WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableViewController.swift

```diff
@@ -101,6 +104,12 @@ extension LoginEpilogueTableViewController {
     }
 
     override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+
+        // Don't show section header for User Info
+        guard section != Sections.userInfoSection else {
+            return nil
+        }
+
         guard let cell = tableView.dequeueReusableHeaderFooterView(withIdentifier: Settings.headerReuseIdentifier) as? EpilogueSectionHeaderFooter else {
             fatalError("Failed to get a section header cell")
         }
```

Comment:
I know this predates the PR, but do we really need to force crash the app if we don't dequeue the correct footer? Maybe we could just fall back and continue with an empty cell, since the login should have already happened?

Reply:
Nice catch! On it!